### ISSUE TITLE
split multipage

### ISSUE THREAD

**Issue by jay982374** [strukturag/libheif#1018]
Hello,

Is there a way in the API to take a single input .heif file with multiple AVIF frames and output one .heif-file per frame without re-compressing each frame ?

**Comment by farindk** [comment]
No, currently not. However, it might be feasible to add such a possibility in the future.
Please leave this issue open as a reminder.

**Comment by bradh** [comment]
Do you have an example of the input AVIF with multiple frames that you can share?
